package.json

```diff
@@ -490,6 +505,15 @@
           "light": "media/changeConnection.svg"
         }
       },
+      {
+        "command": "mssql.executionPlan",
+        "title": "View Execution Plan",
```

Comment:
maybe "Open Execution Plan File" is more precise description of the command behavior.

Reply:
changed it